pkg/snapstore/utils.go

```diff
@@ -179,36 +179,45 @@ func adaptPrefix(snap *brtypes.Snapshot, snapstorePrefix string) string {
 	return snapstorePrefix
 }
 
-// GetSnapstoreSecretHash returns the hash of object store secrets hash
-func GetSnapstoreSecretHash(config *brtypes.SnapstoreConfig) (string, error) {
+// GetSnapstoreSecretModTime returns the latest modification timestamp of the access credential files.
+// Returns an error if fetching the timestamp of the access credential files fails.
+func GetSnapstoreSecretModTime(config *brtypes.SnapstoreConfig) (time.Time, error) {
```

Comment:
```suggestion
func GetSnapstoreSecretModTime(snapStoreProvider string) (time.Time, error) {
```
change the rest of code accordingly